concepts/numbers/about.md

```diff
@@ -0,0 +1,66 @@
+# Haskell Built-in Numeric Types
+
+## Numbers
+
+Haskell's standard library, known as ***Prelude***, defines  most basic numeric types: fixed sized integers (Int), arbitrary precision integers (Integer), single precision floating (Float), and double precision floating (Double). Other numeric types such as rationals and complex numbers are defined in additional libraries.
+
+`Integers` hold any number no matter how big, up to the limit of your machine's memory, while `Int` corresponds to the set of positive and negative integers that can be expressed in 32 or 64 bits (guaranteed at least -2^29 to 2^29).
+
+`Float` corresponds to the set of real numbers, limited by the precision of the computer.
+Operations defined on numbers usually work on one type or the other, but not mixing them.
+
+There are functions to convert between `Float` and `Integer`, such as `toFloat` which converts `Int\Integer` to `Float` and `round` and `ceiling` among others which convert a `Float` to an `Int` with different semantics.
+
+## Arithmetic operators
+
+Haskell has three "raise to the power" operators which work differently and take different argument types.
+
+- `**` Takes two **floating point numbers** and uses logarithms to compute the power.
+
+- `^^` Takes a **floating point** and raises it to a positive or negative **integer** power.
+
+- `^` Takes **any numerical type** and raises it to a **positive integer** power.
+
+Conversion from an integer type (Int or Integer) to anything else is done using "fromIntegral". The target type is inferred automatically. For example:
+
+```haskell 
+n :: Integer
+n = 6
+x :: Float
+x = fromIntegral n      --> x = 6.0
+
+m :: Int
+m = 7
+y :: Double
+y = fromIntegral m      --> y = 7.0
+```
+
+Division of integers is a little complicated. If you use the ordinary "/" operator on integers, then you will get an error message (although the expression "4/3" does work because Haskell helpfully promotes literal integers to floats where necessary). Instead, integer division is done using a collection of named operators.
```

Comment:
Quoting code:

```suggestion
Division of integers is a little complicated. If you use the ordinary `/` operator on integers, then you will get an error message (although the expression `4/3` does work because Haskell helpfully promotes literal integers to floats where necessary). Instead, integer division is done using a collection of named operators.
```